### PR TITLE
Add Code of Conduct Page

### PR DESCRIPTION
**Pull request by ErikaVasNormandy**
- Got the previous code of conduct page from the Way Back Archive
- Copied and pasted the content after reviewing for any broken or out of date links and contact info

### :scroll: Description
this resolves issue  Code of Conduct page missing #75 
We may want to have an email for FAQ that people can reach out to instead of shared-admin

### CODE REVIEW

File: codeofconduct.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+<body>
+            <div class="container">
+
+                <h1>Code of Conduct</h1>
+              
+                <p>This document is adapted from Code for America’s Code of Conduct and Anti-Harrassment Policy.</p>
+                
+                <hr>
+              
+                <h3>Code for San Jose's Code of Conduct</h3>
```
Comment (sunnymui) [inline codeofconduct.html:36]: Do we need 2 titles?

This could probably be the h1 text

File: codeofconduct.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                <p>The Code for San Jose community expects that Code for San Jose network activities, events, and digital forums:</p>
+              
+                <ol>
+                  <li>Are a safe and respectful environment for all participants.</li>
+                  <li>Are a place where people are free to fully express their identities.</li>
+                  <li>Presume the value of others. Everyone’s ideas, skills, and contributions have value.</li>
+                  <li>Don’t assume everyone has the same context, and encourage questions.</li>
+                  <li>Find a way for people to be productive with their skills (technical and not) and energy. Use language such as “yes/and”, not “no/but.”</li>
+                  <li>Encourage members and participants to listen as much as they speak.</li>
+                  <li>Strive to build tools that are open and free technology for public use. Activities that aim to foster public use, not private gain, are prioritized.</li>
```
Comment (sunnymui) [inline codeofconduct.html:47]: Prob don't need `technology` in this sentence if `tools` is there

File: codeofconduct.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                  <li>Prioritize access for and input from those who are traditionally excluded from the civic process.</li>
+                  <li>Work to ensure that the community is well-represented in the planning, design, and implementation of civic tech. This includes encouraging participation from women, minorities, and traditionally marginalized groups.</li>
+                  <li>Actively involve community groups and those with subject matter expertise in the decision-making process.</li>
+                  <li>Ensure that the relationships and conversations between community members, the local government staff and community partners remain respectful, participatory, and productive.</li>
```
Comment (sunnymui) [inline codeofconduct.html:51]: Missing a comma here before the `and`. Should be:

`Ensure that the relationships and conversations between community members, the local government staff, and community partners remain respectful, participatory, and productive.`

File: codeofconduct.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+              
+                <p>Harassment includes but is not limited to: offensive verbal or written comments related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, religion; sexual images in public spaces; deliberate intimidation; stalking; following; harassing photography or recording; sustained disruption of talks or other events; inappropriate physical contact; unwelcome sexual attention; unwarranted exclusion; and patronizing language or action.</p>
+              
+                <p>If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion from Code for San Jose network activities, events, and digital forums.</p>
+               
+                <p>If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact Code for San Jose Captains <a href="mailto:sharedadmin@codeforsanjose.org">on our shared email</a>.</p>
```
Comment (sunnymui) [inline codeofconduct.html:73]: Double checking that this email still works?

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**